text/0001-rfc_process.md

```diff
@@ -0,0 +1,244 @@
+- Feature Name: rfc_process
+- Start Date: 2018-08-13
+- RFC PR: #1
+- Issue: N/A
+
+# Summary
+[summary]: #summary
+
+The "RFC" (request for comments) process is intended to provide a consistent
+and controlled path for charting the roadmap of qri. We've seen a number of 
+projects in the distributed space suffer from under-considered design choices 
+and unclear roadmapping, We're hoping strong adherence to a lightweight RFC 
+process can help mitigate these problems. Anoyone should be able get a sense of 
+_where qri is going_ by reading through the accepted proposals.
+
+# Motivation
+[motivation]: #motivation
+
+I openly adknowledge this may seem premature for such an early-stage project.
+I'm intending to put this RFC place in process now to develop a design-driven
+culture that others have a clear path to contribute to the future of the 
+project, and replace myself as the arbiter of "what qri is" with a codified 
+principle of [rough consensus and working code](https://www.ietf.org/about/participate/tao/).
+
+One of the main concerns I have is ambiguity exists around what is _expected 
+behaviour_ when interacting with qri. To me, this is a _major_ problem. 
+Not knowing precisely what working-as-expected looks like cuts down on what core 
+team members can confidently reason about, which means the core can't
+confidently tell _others_ how qri should work, which means the whole project is
+both ambiguous and without a clear path for resolving that ambiguity.
+
+To me this ambiguity is a sign that we (the core team) haven't taken aren't
```

Comment:
"haven't taken aren't taking" => "haven't taken/aren't taking" or just one or the other?